lib/shared/utils.js

```diff
@@ -81,3 +84,53 @@ exports.makeFlatStartCaseObject = (object) => {
   });
 
 };
+
+/**
+ * @summary Create an object clone with all absolute paths replaced with the path basename
+ * @function
+ * @private
+ *
+ * @param {Object} object - original object
+ * @returns {Object} transformed object
+ *
+ * @example
+ * const anonymized = utils.hidePaths({
+ *   path1: '/home/john/rpi.img',
+ *   simpleProperty: null,
+ *   nested: {
+ *     path2: '/home/john/another-image.img',
+ *     path3: 'yet-another-image.img',
+ *     otherProperty: false
+ *   }
+ * });
+ *
+ * console.log(anonymized);
+ * > {
+ * >   path1: 'rpi.img',
+ * >   simpleProperty: null,
+ * >   nested: {
+ * >     path2: 'another-image.img',
+ * >     path3: 'yet-another-image.img',
+ * >     otherProperty: false
+ * >   }
+ * > }
+ */
+exports.hideObjectAbsolutePaths = (object) => {
+  return _.deepMapValues(object, (value) => {
+    if (!_.isString(value)) {
+      return value;
+    }
+
+    if (os.platform() !== 'win32' && _.startsWith(value, '/dev/')) {
```

Comment:
Yeah, I think we are safe to drop them. This means we can keep a nice array of the paths we check, potentially re-using some logic. For example:

```js
_.each([ 
    '/dev/',
    '\\\\.\\'
], (whitelistedPath) => {
    if (_.startsWith(value, whitelistedPath)) {
        return;
    }
});
```